### DISCUSSION TITLE
"more than one" alert-notification service instance using btpsa usecase

### DISCUSSION THREAD

**Discussion by IdanPeretz** [SAP-archive/btp-setup-automator#296] [Q&A] [answered]
Hi,

I've tried to learn using the btp-setup-automator, and followed the guide using the pre-built image and the default use-case "btpsa".
The flow failed to instantiate alert-notification service, free plan, with the following error: 
`ERROR      : there is more than one service with the instance name >alert-notification-free-instance<. Please fix that before moving on.`

See screenshot below:
![image](https://user-images.githubusercontent.com/37809530/200188890-13e43b4b-92e4-48fe-89d2-c6447dab6249.png)

I am using the "parameters.json" with default values, except the "loginmethod" which I've changed to "sso".
(I am a SAP employee and my cpcli + cfcli user requires OTP with my password).

The Global Account has no alert-notification instance anywhere.

Any idea?

**Comment by rui1610** [comment, marked as answer]
Was an issue with the updated default.json file. Just fixed it and it should be fine now again.
